scripts/attach-static-vdis.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Attach statically-configured VDIs to dom0
+Before=xapi.service
```

Comment:
Not familiar with systemd, would this line be enough to keep the service starting before xapi and also stopping after xapi?